services/graph/pkg/unifiedrole/roles.go

```diff
@@ -109,6 +111,12 @@ var (
 	// UnifiedRole SpaseEditor, Role DisplayName (resolves directly)
 	_spaceEditorUnifiedRoleDisplayName = l10n.Template("Can edit")
 
+	// UnifiedRole SpaseEditorWithoutVersions, Role Description (resolves directly)
+	_spaceEditorWithoutVersionsUnifiedRoleDescription = l10n.Template("View, download, upload, edit, add and delete.")
+
+	// UnifiedRole SpaseEditorWithoutVersions, Role DisplayName (resolves directly)
+	_spaceEditorWithoutVersionsUnifiedRoleDisplayName = l10n.Template("Can edit without versions")
```

Comment:
Just for couriosity: *edit without versions* means that one can edit a file, but no version will be created. It will get plain overwritten. Right?

Reply:
No versions are visible to the share receiver.

Reply:
ok, means, versiones are created, but not visible to the share receiver, correct?
The share receiver only gets/sees the latest available "version".